### PR TITLE
add links to support and slack in issue template

### PR DESCRIPTION
**Pull request by xmulligan**
```release-note
add links to enterprise support and slack to the issues page for easier discoverability
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
```
Comment (bimmlerd) [inline .github/ISSUE_TEMPLATE/config.yml]: Do we really want to encourage _not_ using the issue templates?

**Review by bimmlerd — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (xmulligan) [reply .github/ISSUE_TEMPLATE/config.yml]: ahh sorry that was a copy paste error. Will remove

Comment (xmulligan) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+contact_links:
+  - name: Get Enterprise Support
+    url: https://cilium.io/enterprise/
+    about: See Cilium Enterprise Distributions and Trainings
+  - name: Join Slack
+    url: https://cilium.io/enterprise/
```
Comment (bimmlerd) [inline .github/ISSUE_TEMPLATE/config.yml]: it's the same URL twice, is that intentional?

Comment (xmulligan) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://cilium.io/slack
```

Comment (xmulligan) [reply .github/ISSUE_TEMPLATE/config.yml]: No, I'm sorry 😞 fixed it now

**Review by bimmlerd — APPROVED** [review]
LGTM :)

**Comment by xmulligan** [comment]
This is just a Github issue template change so I'm going to ignore CI and set the ready to merge label

**Review by ldelossa — APPROVED** [review]

**Pull request merged**